addons/html_editor/static/src/main/powerbox/powerbox_plugin.js

```diff
@@ -68,9 +75,33 @@ export class PowerboxPlugin extends Plugin {
             },
             applyCommand: this.applyCommand.bind(this),
         };
-
+        this.defaultPowerboxItems = this.makeDefaultPowerboxItems();
```

Comment:
I renamed them to `powerboxCommands` and `makePowerboxCommands`. I'm using `command` instead of `items` because `items` are the objects declared under the `powerbox_items` resources, whereas `command` are the resulting object after infering properties from `user_command` and the category name from `powerbox_category`.

Do you agree?


```js
/**
 * @typedef {Object} PowerboxItem
 * @property {string} categoryId
 * @property {string} commandId
 * @property {Object} [commandParams]
 * @property {TranslatedString} [title]
 * @property {TranslatedString} [description]
 * @property {string} [icon]
 * @property {TranslatedString[]} [keywords]
 * @property {(selection: EditorSelection) => boolean} [isAvailable] 
 */
/**
 * @typedef {Object} PowerboxCommand
 * @property {string} categoryId
 * @property {string} categoryName
 * @property {string} title
 * @property {string} description
 * @property {string} icon
 * @property {Function} run
 * @property {TranslatedString[]} [keywords]
 * @property { (selection: EditorSelection) => boolean  } [isAvailable]
 */

```